tests/func/experiments/test_show.py

```diff
@@ -530,26 +553,26 @@ def _get_rev_isotimestamp(rev):
     capsys.readouterr()
     assert main(["exp", "show", "--csv"]) == 0
     cap = capsys.readouterr()
+    data_dep = first(x for x in dvc.index.deps if "copy.py" in x.fspath)
+    data_hash = data_dep.hash_info.value[:7]
```

Comment:
Didn't know how to something like the `ANY` usage above but for the type of assertions bellow